packages/fork-choice/src/metrics.ts

```diff
@@ -0,0 +1,22 @@
+import {MetricsRegister} from "@lodestar/utils";
+import {FCInclusionListSource} from "./index.js";
+
+export type BeaconForkChoiceMetrics = ReturnType<typeof getForkChoiceMetrics>;
+
+/**
+ * A collection of metrics used throughout the Fork Choice.
+ */
+export function getForkChoiceMetrics(register: MetricsRegister) {
```

Comment:
we already have fork choice metrics here

https://github.com/ChainSafe/lodestar/blob/f3e21c76794869cadedccce10ca02401e150351a/packages/beacon-node/src/metrics/metrics/beacon.ts#L76

might be best to not define them in two places now and move them here in a separate refactoring